gorm_bulk.go

```diff
@@ -139,7 +130,14 @@ func scopeFromObjects(db *gorm.DB, objects []interface{}, execFunc ExecFunc) (*g
 		}
 
 		for _, key := range columnNames {
-			objectScope.AddToVars(row[key])
+			// Column CreatedAt and UpdatedAt with zero value will be set to same time
+			if name := row[key].Struct.Name; name == "CreatedAt" || name == "UpdatedAt" {
+				if t, ok := row[key].Field.Interface().(time.Time); ok && t.IsZero() {
+					_ = row[key].Set(bulkNow)
```

Comment:
But this may cause the different `CreatedAt` or  `UpdatedAt` for rows in the same `BulkInsert`. To make them consistent, we should get the value at the beginning of func `scopeFromObjects`.

I think we should remove the comment but keep the variable.

Reply:
You’re right, I’m stupid. That was the whole idea behind this in the first place. :)

Keep the variable, dump the comment. Maaaybe keep s comment about why it’s there, like *nowFunc* is stored first to result in the same time for every record* or whatever. You can decide if and what! :)